lib/widget/camera.dart

```diff
@@ -48,9 +48,18 @@ class CameraScreen extends StatefulWidget
           Utils.optionalBool(value) ?? _controller.preview,
       'maxCountMessage': (value) =>
           _controller.maxCountMessage = Utils.getString(value, fallback: ''),
+      'permissionDeniedMessage': (value) => _controller
```

Comment:
@hamxa152 all fallback strings should be the default strings and not empty strings. Please do that for all fallback strings